protocol/answer/answer.json

```diff
@@ -5,28 +5,7 @@
 	"description": "Match a positive or negative server answer",
 	"type": "object",
 	"properties": {
-		"jsonrpc": {
-			"description": "[String] JsonRpc version",
-			"const": "2.0",
-			"$comment": "should always be \"2.0\""
-		},
-		"result": {
-			"description": "In case of positive answer, result of the client query",
-			"oneOf": [
-				{
-					"$ref": "positiveAnswer/positiveAnswerGeneral.json"
-				},
-				{
-					"$ref": "positiveAnswer/positiveAnswerCatchup.json"
-				}
-			],
-			"$comment": "Note: this field is absent if the answer is negative"
-		},
-		"error": {
-			"description": "In case of negative answer, error generated by the client query",
-			"$ref": "negativeAnswer/error.json"
-		},
-		"id": {
+		"requestID": {
```

Comment:
`id` is mandated by the JSON-RPC standard.
https://www.jsonrpc.org/specification